src/features/types/faq-schema.ts

```diff
@@ -0,0 +1,18 @@
+type contentType = 'FAQPage' | 'Question' | 'Answer'
+
+type AcceptedAnswerType = {
+    '@type': contentType
+    text: string
+}
+
+type MainEntityType = {
```

Comment:
hi man  good move can you update this in accumulated schema too, if you don't mind, please

Reply:
hi man, it's updated in accumulator already. plus, updated the "How do accumulator options work" tag to h2